replay_pid29889.log

```diff
@@ -0,0 +1,2506 @@
+version 2
```

Comment:
The same here - or I missed something, or it should be also deleted, what do you think?

Reply:
Thanks for noticing it. Removed :)